### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by viktoriazakatsyura** [community/community#27959] [New to GitHub]
Hello,
Have an error with installing.
![Знімок екрана 2022-08-01 235450](https://user-images.githubusercontent.com/109316794/182245115-89533590-3a23-4df1-9d90-2e31175e95d8.png).
What to do with this?

**Comment by jfullstackdev** [comment]
what is the operating system ? Windows 10 ? do you have 32-bit or 64-bit OS ? I just installed GitHub Desktop, there are no errors

**Comment by viktoriazakatsyura** [comment]
Intel - Windows 10 - 64bit

вт, 2 серп. 2022 р. о 07:06 jdevstatic ***@***.***> пише:
